src/Elmish.WPF/Binding.fs

```diff
@@ -131,6 +131,18 @@ and Binding<'model, 'msg> =
 
 module internal BindingData =
 
+
+  let subModelSelectedItemDataLast a b =
+    match a, b with
+    | SubModelSelectedItemData _, SubModelSelectedItemData _ -> 0
+    | SubModelSelectedItemData _,                          _ -> 1
+    |                          _, SubModelSelectedItemData _ -> -1
+    |                          _,                          _ -> 0
```

Comment:
I see why you'd align it like this, but personally I never do because it's unstable if things are renamed. I'd just do this:

```f#
    match a, b with
    | SubModelSelectedItemData _, SubModelSelectedItemData _ -> 0
    | SubModelSelectedItemData _, _ -> 1
    | _, SubModelSelectedItemData _ -> -1
    | _, _ -> 0
```

But if you prefer it aligned, that's fine with me.

Reply:
Sure, removed the white space.  I only have a mild preference for having things aligned.